cronJob/cronJob.js

```diff
@@ -1,58 +1,56 @@
 const cron = require('node-cron');
-const subscriptionController = require('../controllers/subscriptionController');
 const request = require('request');
 const moment = require("moment");
-const {format} = require("winston");
 const {Subscription} = require('../models/subscriptions')
+const serverLogger = require(`../logger`);
+const sendQueueUpdate = require("../RMQ/rabbitMQ");
 
-require("dotenv").config({path: '.env'});
+const logger = serverLogger.log;
 
-const addMonthForDate =(subscription) => {
+
+const updateSubscription = (subscription, credits) => {
 
     let nextMonth = moment().add(1, 'M').format('YYYY-MM-DD HH:mm:ss');
```

Comment:
if you use this pattern `'YYYY-MM-DD HH:mm:ss'` in other places please consider moving it into a constant